src/main/java/com/kodilla/ecommercee/mapper/OrderMapper.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.ecommercee.mapper;
+
+import com.kodilla.ecommercee.domain.Order;
+import com.kodilla.ecommercee.domain.OrderDto;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class OrderMapper {
+    public Order mapToOrder(final OrderDto orderDto) {
+        return new Order(
+                orderDto.getId(),
+                orderDto.getProduct(),
```

Comment:
W OrderDto powinno być tylko product_id, nie cały obiekt Product. Przy tworzeniu obiektu domenowego z Dto trzeba to pole uzupełnić przez productService wyciągając obiekt na podstawie jego id.